dags/sources/tasks/business_logic/source_data_normalize.py

```diff
@@ -187,35 +186,30 @@ def source_data_normalize(
 
     # TODO: Remplacer par le dag_id
     if dag_id == "sinoe":
-        df = df_normalize_sinoe(
-            df,
-            product_mapping=dag_config.product_mapping,
-            dechet_mapping=dag_config.dechet_mapping,
-        )
+        df = df_normalize_sinoe(df)
 
     # Merge et suppression des lignes indésirables
     df = _remove_undesired_lines(df, dag_config)
 
+    # TODO: A voir ce qu'on doit faire de ces acteurs non digitaux mais sans
+    # localisation (proposition : les afficher en erreur directement ?)
+    if "location" in df.columns and "acteur_type_code" in df.columns:
+        df_acteur_sans_loc = df[
+            (df["location"].isnull()) & (df["acteur_type_code"] != "acteur_digital")
+        ]
+        if not df_acteur_sans_loc.empty:
+            nb_acteurs = len(df)
+            logger.warning(
+                f"Nombre d'acteur sans localisation: {len(df_acteur_sans_loc)} / "
+                f"{nb_acteurs}"
+            )
+            log.preview("Acteurs sans localisation", df_acteur_sans_loc)
+
```

Comment:
à voir si utile d'isoler dans une fonction qui explique ce qui est fait 

```suggestion
    display_warning_about_missing_location(df, log)
```